frame/society/src/lib.rs

```diff
@@ -1081,188 +1143,194 @@ pub mod pallet {
 			Self::deposit_event(Event::<T, I>::SuspendedMemberJudgement { who, judged: forgive });
-			Ok(())
+			Ok(Pays::No.into())
 		}
 
-		/// Allow suspended judgement origin to make judgement on a suspended candidate.
+		/// Change the maximum number of members in society and the maximum number of new candidates
+		/// in a single intake period.
 		///
-		/// If the judgement is `Approve`, we add them to society as a member with the appropriate
-		/// payment for joining society.
-		///
-		/// If the judgement is `Reject`, we either slash the deposit of the bid, giving it back
-		/// to the society treasury, or we ban the voucher from vouching again.
-		///
-		/// If the judgement is `Rebid`, we put the candidate back in the bid pool and let them go
-		/// through the induction process again.
-		///
-		/// The dispatch origin for this call must be from the _SuspensionJudgementOrigin_.
+		/// The dispatch origin for this call must be Signed by the Founder.
 		///
 		/// Parameters:
-		/// - `who` - The suspended candidate to be judged.
-		/// - `judgement` - `Approve`, `Reject`, or `Rebid`.
+		/// - `max_members` - The maximum number of members for the society. This must be no
+		///   less than the current number of members.
+		/// - `max_intake` - The maximum number of candidates per intake period.
+		/// - `max_strikes`: The maximum number of strikes a member may get before they become
+		///   suspended and may only be reinstated by the founder.
+		/// - `candidate_deposit`: The deposit required to make a bid for membership of the group.
 		///
-		/// # <weight>
-		/// Key: B (len of bids), M (len of members), X (balance action)
-		/// - One storage read to check `who` is a suspended candidate.
-		/// - One storage removal of the suspended candidate.
-		/// - Approve Logic
-		/// 	- One storage read to get the available pot to pay users with. O(1)
-		/// 	- One storage write to update the available pot. O(1)
-		/// 	- One storage read to get the current block number. O(1)
-		/// 	- One storage read to get all members. O(M)
-		/// 	- Up to one unreserve currency action.
-		/// 	- Up to two new storage writes to payouts.
-		/// 	- Up to one storage write with O(log M) binary search to add a member to society.
-		/// - Reject Logic
-		/// 	- Up to one repatriate reserved currency action. O(X)
-		/// 	- Up to one storage write to ban the vouching member from vouching again.
-		/// - Rebid Logic
-		/// 	- Storage mutate with O(log B) binary search to place the user back into bids.
-		/// - Up to one additional event if unvouch takes place.
-		/// - One storage removal.
-		/// - One event for the judgement.
-		///
-		/// Total Complexity: O(M + logM + B + X)
-		/// # </weight>
+		/// Total Complexity: O(1)
 		#[pallet::weight(T::BlockWeights::get().max_block / 10)]
-		pub fn judge_suspended_candidate(
+		pub fn set_parameters(
 			origin: OriginFor<T>,
-			who: T::AccountId,
-			judgement: Judgement,
+			max_members: u32,
```

Comment:
Could use something like this for easier updates: https://github.com/paritytech/substrate/blob/cfb5bc6f1a92207785a0bdd321039096685623c8/frame/staking/src/pallet/mod.rs#L66